scripts/globals/items/pot_of_silent_oil.lua

```diff
@@ -13,7 +13,7 @@ item_object.onItemCheck = function(target)
 end
 
 item_object.onItemUse = function(target)
-    local duration = math.random(300, 480)
+    local duration = (600)
```

Comment:
1. parenthesis not needed
2. 2 now that its not random, it doesn't even need variablized

Reply:
same in other places